app/javascript/components/visualization/InferCNVIdeogram.js

```diff
@@ -0,0 +1,300 @@
+    window.originalHeatmapThresholds =
+      inferCNVIdeogram.rawAnnots.metadata.heatmapThresholds
+  }
+
+  const expressionThresholdSlider =
+    `<label id="expressionThresholdContainer" style="float: left">
+      <span
+        class="glossary"
+        title="Denoiser.  Adjusts mapping between inferCNV's output heatmap
+          threshold values and normal vs. loss/gain signal.  Analogous to
+          inferCNV denoise parameters, e.g. --noise_filter."
+        style="cursor: help;">
+      Expression threshold
+      </span>
+      <input
+        type="range"
+        id="expressionThreshold"
+        list="expressionThresholdList"
+        value="${expressionThreshold}"
+      >
+      <datalist id="expressionThresholdList">
+        <option value="0" label="0.">
+        <option value="10">
+        <option value="20">
+        <option value="30">
+        <option value="40">
+        <option value="50" label="1">
+        <option value="60">
+        <option value="70">
+        <option value="80">
+        <option value="90">
+        <option value="100" label="1.5">
+      </datalist>
+      <br/><br/>`
+  d3.select('#_ideogramLegend').node().innerHTML += expressionThresholdSlider
+}
+
+/** Handle updates to slider controls for ideogram display */
+function ideoRangeChangeEventHandler(event) {
+  const id = event.target.id
+  if (id === 'expressionThreshold') updateThreshold(event)
+  if (id === 'chrMargin') updateMargin(event)
+}
+
+/** Add sliders to adjust ideogram display */
+function addIdeoRangeControls() {
+  addThresholdControl()
+  addMarginControl()
+
+  document.removeEventListener('change', ideoRangeChangeEventHandler)
+  document.addEventListener('change', ideoRangeChangeEventHandler)
+}
+
+/** Create interactive filters for ideogram tracks */
+function createTrackFilters() {
+  let i; let listItems; let checked
+
+  addIdeoRangeControls()
+
+  // Only apply this function once
+  if (document.querySelector('#filter_1')) return
+  listItems = ''
+  const trackLabels = inferCNVIdeogram.rawAnnots.keys.slice(6)
+  const displayedTracks = inferCNVIdeogram.config.annotationsDisplayedTracks
+  for (i = 0; i < trackLabels.length; i++) {
+    checked = (displayedTracks.includes(i + 1)) ? 'checked' : ''
+    listItems +=
+      `${'<li>' +
+      '<label for="filter_'}${i + 1}">` +
+      `<input type="checkbox" id="filter_${i + 1}" ${checked}/>${
+        trackLabels[i]
+      }</label>` +
+      `</li>`
+  }
+  const content = `Tracks ${listItems}`
+  document.querySelector('#tracks-to-display').innerHTML = content
+
+
+  $('#filters-container').after(
+    '<div id="ideogramTitle">Copy number variation inference</div>'
+  )
+
+  checkboxes = document.querySelectorAll('input[type=checkbox]')
+  checkboxes.forEach(checkbox => {
+    checkbox.addEventListener('click', () => {
+      updateTracks()
+    })
+  })
+}
+
+/**
+ * Note ideogram is unavailable for this numeric cluster
+ *
+ * Used in render_cluster.js.erb
```

Comment:
Remove obsolete part of comment.

```suggestion
 * Note ideogram is unavailable for this numeric cluster
```